report/models.go

```diff
@@ -0,0 +1,20 @@
+package report
+
+import (
+	"time"
+)
+
+const(
+	ReportCollectionName  = "reports" 
+)
+
+type Report struct {
+	ID                string		`bson:"_id,omitempty" json:"_id,omitempty"`
+	ReporterName      string        `bson:"reporter_name" validate:"required,min=2,max=100" json:"reporter_name"`
+	ReporteeName      string        `bson:"reportee_name" validate:"required,min=2,max=100" json:"reportee_name"`
+	Organization      string        `bson:"organization" validate:"required" json:"organization"`
```

Comment:
You didn't use this field. It should serve as a foreign key in report collections. That way we can getalldocs filtered by organization just like we did on members. So for uniqueness, it should be an organization Id. We may end up moving this into the organization folder, maybe in its separate file. the endpoint will look like: 
organizations/id/reports - POST - to create a report
organizations/id/reports - GET - to fetch all reports for an organization 
organizations/id/reports/id - GET, DELETE, PUT on a report